fastcrypto-zkp/src/verifier.rs

```diff
@@ -67,7 +67,7 @@ const G1_IDENTITY: blst_p1 = blst_p1 {
 };
 
 /// Returns the log base 2 of b in O(lg(N)) time.
-pub fn log_2_byte(b: u8) -> usize {
+fn log_2_byte(b: u8) -> usize {
```

Comment:
In the future we should move this to a common math utils file, potentially reusable among all fastcrypto crates. No action in this PR.